src/AppBundle/Entity/User.php

```diff
@@ -92,6 +126,9 @@ public function getRoles()
         return array_unique($roles);
     }
 
+    /**
+     * @param $roles
```

Comment:
param type? should be "@param array $roles" or "@param Role[] $roles" if its array of type Role i.e.